src/interpreter.rs

```diff
@@ -910,13 +927,13 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
                 try!(self.memory.write_bool(ptr, b));
                 Ok(ptr)
             }
-            Char(_c)          => unimplemented!(),
-            Struct(_node_id)  => unimplemented!(),
-            Tuple(_node_id)   => unimplemented!(),
+            Char(_c) => unimplemented!(),
+            Struct(_node_id) => unimplemented!(),
+            Tuple(_node_id) => unimplemented!(),
             Function(_def_id) => unimplemented!(),
-            Array(_, _)       => unimplemented!(),
-            Repeat(_, _)      => unimplemented!(),
-            Dummy             => unimplemented!(),
+            Array(_, _) => unimplemented!(),
+            Repeat(_, _) => unimplemented!(),
+            Dummy => unimplemented!(),
```

Comment:
... and here's one place I *do* use alignment and rustfmt doesn't. But I don't really care about this.